wpeverest/everest-forms
# Enhancement - Screenshot video on upgrade to pro popup

### All Submissions:


### Changes proposed in this Pull Request:

This PR is introducing the video section in the upgrade pro popup.

### How to test the changes in this Pull Request:

1. Active the Everest form free.
2. Click on the locked field and verify the videos in all fields

### Types of changes:


### Other information:



### Changelog entry

> Enhancement - Screenshot video on upgrade to pro popup.
